### PR TITLE
Docker Obiba Drupal Portal

### PR DESCRIPTION
**Pull request by smirfolio**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 Docker Obiba Drupal Portal
 ===========
 
-Launch obiba Drupal POrtal container (see Makefile for more details):
+Launch obiba Drupal Mica Portal container (see Makefile for more details):
```
Comment (ymarcon) [inline README.md]: I do not see anything about mica in this docker.

**Comment by ymarcon** [comment]
I still do not understand why this stuff is not in the docker-mica-drupal repo.

**Comment by smirfolio** [comment]
It's related to https://jira.obiba.org/jira/browse/MK-1750
By default it query the local Mica server, but once lunched it can be linked to any Mica server (Exp: https://nightly-mica.obiba.org)

**Comment by ymarcon** [comment]
If your intent is to make a docker that contains drupal configured with mica modules, use the docker-mica-drupal repository.

**Pull request closed without merging**